string behavior_name
---
bool result
string[] requirements
